content/en/security/code_security/iast/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: IAST
+disable_toc: false
+aliases:
+- /path-to-old-doc/
+---
+
+## Overview
+
+Datadog Code Security identifies code-level vulnerabilities in your services and provides actionable insights and recommended fixes.
+
+For a list of supported services, see the [Library Compatibility Requirements][5].
+
+Code Security uses an Interactive Application Security Testing (IAST) approach to find vulnerabilities within your application code. IAST uses instrumentation embedded in your code like application performance monitoring (APM).
+
+Code Security also monitors your code’s interactions with other components of your stack, such as libraries and infrastructure.
+
+IAST enables Datadog to identify vulnerabilities using legitimate application traffic instead of relying on external tests that could require extra configuration or periodic scheduling.
```

Comment:
```suggestion
IAST enables Datadog to identify vulnerabilities using legitimate application traffic instead of relying on external tests that could require extra configuration or periodic scheduling. It also monitors your code’s interactions with other components of your stack, such as libraries and infrastructure, providing an up-to-date view of your attack surface area.
```